pkg/runtime/ast/create_index.go

```diff
@@ -38,6 +50,17 @@ func (c *CreateIndexStatement) CntParams() int {
 
 func (c *CreateIndexStatement) Restore(flag RestoreFlag, sb *strings.Builder, args *[]int) error {
 	sb.WriteString("CREATE INDEX ")
```

Comment:
Should remove `INDEX`.

Reply:
Thanks for the review, new commits have modified these.